app/assets/stylesheets/application.scss

```diff
@@ -30,6 +30,27 @@
  //   -ms-transform: translateY(-50%);
  //   transform: translateY(-50%);
  // }
+
+ .green-completion{
+  color: white;
+  width: 85px;
+  background-color: green;
+  margin: 5px 5px 5px 5px;
+  border-width: 1px;
+  border-style: solid;
+  border-color: rgba(0,0,0,.2);
+ }
+
+.red-completion {
+  color: white;
+  width: 45px;
+  background-color: red;
+  margin: 5px 5px 5px 5px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `5px` instead of `5px 5px 5px 5px`